app/src/main/res/values/strings.xml

```diff
@@ -355,7 +355,7 @@
     <string name="github_url" translatable="false">https://github.com/TeamNewPipe/NewPipe</string>
     <string name="view_on_github">View on GitHub</string>
     <string name="donation_title">Donate</string>
-    <string name="donation_encouragement">NewPipe is developed by volunteers spending time bringing you the best experience. Give back to help developers make NewPipe even better while enjoying a cup of coffee.</string>
+    <string name="donation_encouragement">Treat the newPipe makers to something you like, by way of funding its development.</string>
```

Comment:
Please use @cool-student's version from https://github.com/TeamNewPipe/NewPipe/pull/2938#discussion_r364598892.

Reply:
Done. However it adds two burdens on the user. One is the implied debt of having already received the software. That may work, but I don't like it.
The second is that of reading that long line of text. If done, that incurs an investment of time, which I also don't like, and really grinds my goat. It is not elegant. https://github.com/TeamNewPipe/NewPipe/pull/2938#discussion_r376792572